tests/common/snappi/common_helpers.py

```diff
@@ -359,11 +358,13 @@ def pfc_class_enable_vector(prio_list):
     return vector
 
 
-def get_wred_profiles(host_ans):
+def get_wred_profiles(host_ans, asic_value="None"):
```

Comment:
Should `asic_value` be called `asic_namespace` or just `namespace` instead?

Reply:
asic_value is more understandable for user perspective rather than using namespace

Reply:
Should the default value be the `None` object instead of the string "None"?

Reply:
@selldinesh  I agree with @kenneth-arista , the default value of asic_value should be None rather than "None"